internal/cmd/schema-registry/command_config.go

```diff
@@ -1,33 +1,23 @@
 package schemaregistry
 
 import (
-	srsdk "github.com/confluentinc/schema-registry-sdk-go"
 	"github.com/spf13/cobra"
 
 	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
 	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
 )
 
-type configCommand struct {
-	*pcmd.AuthenticatedStateFlagCommand
-	srClient *srsdk.APIClient
-}
-
-func newConfigCommand(cfg *v1.Config, prerunner pcmd.PreRunner, srClient *srsdk.APIClient) *cobra.Command {
+func (c *command) newConfigCommand(cfg *v1.Config) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:         "config",
 		Short:       "Manage Schema Registry configuration.",
 		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLoginOrOnPremLogin},
 	}
 
-	c := &configCommand{srClient: srClient}
-
 	if cfg.IsCloudLogin() {
-		c.AuthenticatedStateFlagCommand = pcmd.NewAuthenticatedStateFlagCommand(cmd, prerunner)
-		c.AddCommand(c.newDescribeCommand())
+		cmd.AddCommand(c.newConfigDescribeCommand())
 	} else {
-		c.AuthenticatedStateFlagCommand = pcmd.NewAuthenticatedWithMDSStateFlagCommand(cmd, prerunner)
```

Comment:
this no longer needed?

Reply:
It's only called once by the top-level `confluent schema-registry` command